Gemfile.lock

```diff
@@ -14,7 +14,7 @@ PATH
       rspec (~> 3.2.0)
 
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```

Comment:
Isn't this a regression?  bundler explicitly deprecates http here, or at least it used to.